django_oci/views/blobs.py

```diff
@@ -28,10 +28,20 @@
 from django_oci.utils import parse_content_range
 from django_oci.auth import is_authenticated
 
+from ratelimit.mixins import RatelimitMixin
 
-class BlobDownload(APIView):
+
+class BlobDownload(RatelimitMixin, APIView):
     """Given a GET request for a blob, stream the blob."""
 
+    ratelimit_key = "ip"
+    ratelimit_rate = settings.VIEW_RATE_LIMIT
+    ratelimit_block = settings.VIEW_RATE_LIMIT_BLOCK
```

Comment:
What do you think about porting it to a separate mixin, since this part repeats itself every time it is used in this app?

Reply:
Sure, could definitely do that!

Reply:
Looks like RatelimitMixin is deprecated in favor of method_decorator using it https://github.com/jsocol/django-ratelimit/blob/cc1c3d707a53c4a3035fa7323bceabda125fa68c/CHANGELOG#L23, so probably both approaches here won't work and we'll need to stick with the method decorator.

Reply:
The other option is to reproduce this function and just hard code the defaults, that will be less flexible to changes in the module, but should be ok if we pin to version 3.0.0. It would save a few lines for each decorator https://github.com/jsocol/django-ratelimit/blob/main/ratelimit/decorators.py.